php/README.md

```diff
@@ -0,0 +1,24 @@
+Clever Instant Login PHP Example
+==========================
+
+This sample code demonstrates how to perform token exchange with Clever's Instant Login API.
+
+This example was built using procedural PHP. The sample code itself relies on no external PHP dependencies.
+
+### Running the sample code
+The sample code relies on some environment variables to easily set state. Productionized applications should use more robust configuration schemes.
+
+Use `bin/start_sample_server.sh` to easily start serving on localhost:2587.
+
+`CLEVER_CLIENT_ID=abc CLEVER_CLIENT_SECRET=xyz DISTRICT_ID=hjkl ./bin/start_sample_server.sh`
+
```

Comment:
Why dont we add information about what version(s) of PHP are supported? I'd suggest 5.4+ which is almost 3 years old.  But then we should test with them.